api/routes/subcriterias/get_sub_criteria.py

```diff
@@ -171,6 +171,7 @@ def deprecated_sort_add_another_component_contents(
         "RadioField": "text",
         "textField": "text",
         "MonthYearField": "monthYearField",
+        "monthYearField": "monthYearField",
```

Comment:
Is this duplicating above above field?

Reply:
yes, it was sensitive to the case, so i specified support for both cases. This has now been superseded by matching without case sensitivity in the function, see https://github.com/communitiesuk/funding-service-design-assessment-store/commit/a5ad86ce68d39ca9d4c2c1a07a6cd1b0f5f12b7f